test/services/concerns/xccdf/profiles_test.rb

```diff
@@ -36,5 +37,19 @@ def report_description
         save_profiles
       end
     end
+
+    test 'correctly save value_overrides' do
+      save_profiles
+      profile1 = Profile.find_by(ref_id: 'xccdf_org.ssgproject.content_profile_standard')
+      profile2 = Profile.where(value_overrides: {}).first
+      expected_overrides = { 'xccdf_org.ssgproject.content_value_var_auditd_num_logs' => '5',
+                             'xccdf_org.ssgproject.content_value_sshd_idle_timeout_value' => '300',
+                             'xccdf_org.ssgproject.content_value_var_accounts_maximum_age_login_defs' => '90',
+                             'xccdf_org.ssgproject.content_value_var_accounts_minimum_age_login_defs' => '7',
+                             'xccdf_org.ssgproject.content_value_var_accounts_password_minlen_login_defs' => '12',
+                             'xccdf_org.ssgproject.content_value_var_accounts_password_warn_age_login_defs' => '7' }
+      assert_equal expected_overrides, profile1.value_overrides
+      assert_equal 0, profile2.value_overrides.length
```

Comment:
Please explain the significance of `profile2` in this test, because it does not make sense to me.

Reply:
It was supposed to test a profile that has no value refinements so `value_overrides` should be `{}` but I just realized it doesn't actually make sense the way that I defined `profile2` 🤔 But if I update it to `profile2 = Profile.find_by(ref_id: profile_with_no_refinements_ref_id)` that should work

Reply:
As far as I understand, you search for a profile that has no value overrides and then assert if the number of value overrides for that profile is zero. Sorry, but I fail to understand how does that make any sense.

Reply:
Right, it didn't make sense how I had it before. I updated it so that it's finding a specific Profile from the test file that has no `refine-value` elements:

`profile2 = Profile.find_by(ref_id: 'xccdf_org.ssgproject.content_profile_standard')`

And then verifying that `profile2.value_overrides = {} `